### PR TITLE
[OneExplorer] Show .circle.log file

### PR DESCRIPTION
**Pull request by dayo09**
This commit shows .circle.log file under the cfg file.

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
To me, it seems to have both refactoring and showing the file.

**Comment by YongseopKim** [comment]
So it makes reviewers hard to review, IHMO.

**Comment by YongseopKim** [comment]
Questions about `artifact`

There are the following files
- .tflite
- .circle
- .circle.log

To me, `artifact` could be one and include all of the above files. However, this PR show 3 artifacts to me.

What do I miss? Could you describe it?

File: src/OneExplorer/ConfigObject.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import * as fs from 'fs';
 import * as ini from 'ini';
 import * as path from 'path';
 import * as vscode from 'vscode';
+import {Logger} from '../Utils/Logger';
 
 /**
```
Comment (dayo09) [inline src/OneExplorer/ConfigObject.ts:23]: @YongseopKim I defined the term `Artifact` here.

Comment (dayo09) [reply src/OneExplorer/ConfigObject.ts:23]: Most of times, `artifacts` are a zip file including all the result output files, but here, artifact means each result file, I think that's what you are talking about... I took a reference [here](https://en.wikipedia.org/wiki/Artifact_(software_development)), wikipedia, it doens't mention the notion as  'a set of results files', so I assumed that the term artifact can be used to each file.

> The term artifact in connection with software development is largely associated with specific development methods or processes e.g., [Unified Process](https://en.wikipedia.org/wiki/Unified_Process). This usage of the term may have originated with those methods.

**Comment by dayo09** [comment]
> So it makes reviewers hard to review, IHMO.

Sure.. I will separate theem.

**Comment by dayo09** [comment]
Hold until #1006 is merged

**Comment by dayo09** [comment]
@Samsung/one-vscode PTAL :-D

**Review by YongseopKim — APPROVED** [review]
LGTM 👍

**Comment by dayo09** [comment]
@one-vscode PTAL :-D

**Review by yunjayh — APPROVED** [review]
LGTM

**Comment by YongseopKim** [comment]
ping @llFreetimell

**Pull request merged**